lib/Conversion/TritonGPUToLLVM/PrintOpToLLVM.cpp

```diff
@@ -196,9 +196,6 @@ struct PrintOpConversion : public ConvertOpToLLVMPattern<triton::PrintOp> {
     std::string prefix = "%";
     if (width.has_value()) {
       prefix += std::to_string(*width);
-    } else if (hex) {
```

Comment:
Processing of `hex==True` occurs on line 185, after which `return` statement follows.